packages/syft/src/syft/core/node/common/node_service/task_submission/task_submission.py

```diff
@@ -217,25 +226,126 @@ def run(  # type: ignore
             search_params={TASK_SERVICE_FIELDS.UID.value: self.payload.task_uid},
             updated_args=update_values,
         )
+
         task = node.tasks.find(search_params={"uid": self.payload.task_uid})
         if not task:
             raise Exception(f"The given task_id:{self.payload.task_uid} does not exist")
         task = task[0]
 
+        init_state = init_policy_state(task.policy_code, task.policy_name)
+        # TODO use our better serialization for the state variable
+
+        node.tasks.update(
```

Comment:
Is this the deprecated API?

Reply:
yes, I will eliminate it, I was not sure if there was anyone using that file, but I guess this is not he case anymore